docs/themes/thxvscode/layouts/partials/apiNav.html

```diff
@@ -67,7 +67,7 @@
             {{- end -}}
         {{- end }}
 
-        <hr/>
+        <li><hr/></li>
```

Comment:
this isn't the best solution in that screen readers will see this as a list item with nothing inside of it. Best solution would be to add the gap and horizontal line with CSS alone. 

I wouldn't consider it deal breaking for this PR (better than nothing) but I would verify that we aren't creating another a11y bug

Reply:
I'll fix this with CSS.

Reply:
I addressed this by making a second `ul`.